src/GraphQL/GraphQLBuilderExtensions.cs

```diff
@@ -1029,13 +1029,52 @@ public static IGraphQLBuilder AddValidationRule<TValidationRule>(this IGraphQLBu
         }
         #endregion
 
-        #region - AddMetrics -
+        #region - AddApolloTracing / AddMetrics -
+        /// <summary>
+        /// Registers <see cref="InstrumentFieldsMiddleware"/> within the dependency injection framework and
+        /// configures it to be installed within the schema, and configures responses to include Apollo
+        /// Tracing data when enabled via <see cref="ExecutionOptions.EnableMetrics"/>.
+        /// When <paramref name="enable"/> is <see langword="true"/>, configures execution to set
+        /// <see cref="ExecutionOptions.EnableMetrics"/> to <see langword="true"/>; otherwise leaves it unchanged.
+        /// </summary>
+        public static IGraphQLBuilder AddApolloTracing(this IGraphQLBuilder builder, bool enable = true)
```

Comment:
I'm a bit lean towards `collectMetrics` vs `enable` but I'm OK to leave `enable`.

Reply:
I'm rather indifferent.  `enable` was used with `AddMetrics` to match up to `EnableMetrics`.  Perhaps this should be `enableMetrics` to more clearly reflect that it is setting the `EnableMetrics` option.

Reply:
Consider that, and then let me know your preference and I can change it as you feel would be best.

Reply:
`enebleMetrics` then